src/morph_server/src/Response.re

```diff
@@ -0,0 +1,29 @@
+type Morph.Response.body +=
+  | Stream(Lwt_stream.t(char))
+  | StringStream(Lwt_stream.t(string));
+
+let byte_stream = (stream, res) => {
+  Morph.Response.set_body(Stream(stream), res) |> Lwt.return;
+};
+
+let get_file_stream = file_name => {
+  let read_only_flags = [Unix.O_RDONLY];
+  let read_only_perm = 444;
+  let fd = Unix.openfile(file_name, read_only_flags, read_only_perm);
+  Lwt_io.of_unix_fd(fd, ~mode=Lwt_io.Input) |> Lwt_io.read_chars;
+};
+
+let static = (file_path, res) => {
```

Comment:
I must say that the `Morph_server` module feels `OCaml`y in the a sense. If it's something like a standard library for `Morph` how about naming it `MorphStd`/`Morph_std`?

Reply:
I thought about Morph_server_core since it's shared by the http and https servers currently. This is also the abstraction over httpaf and h2.

Reply:
`Morph_core` maybe? The thing about including `server` in the name is that it makes it easy to confuse with `morph_server_http(s)`. And I like the names `Morph_core` or `Morph_std` (to be honest the latter more but the former is also nice) because it's something you will use even if you use other server implementations. Also... maybe it makes sense to put it in `Morph` itself actually? Less top level modules is good IMO and it doesn't sacrifice composability. On the contrary Morph_core or Morph_std are better names if it's supposed to be extended.

Reply:
I think we can move this to an issue, it's not a blocker for this PR imo. But it would be nice with a more coherent set of modules that are more obvious.

Maybe something like this:
Morph - the core types, middlewares, server and the String body variant. This is the minimum to implement a Morph server
Morph_base/core/std - more types like streaming and helpers for that, maybe some more opinions
Morph_server_http(s) - a actual server implementation
Morph_server_core - a abstraction over httpaf and h2 used by the 2 current server implementations, this is not seen by the enduser

Reply:
I like this plan.

Reply:
I started this but this should probably continue in a PR further down the line